src/pages/_app.tsx

```diff
@@ -115,7 +115,7 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
         <link rel="manifest" href="manifest.json" />
       </Head>
 
-      <Script id="phosphor-icons" src="https://unpkg.com/@phosphor-icons/web@2.0.3" async />
+      <Script id="phosphor-icons" src="https://unpkg.com/@phosphor-icons/web@2.1.0" async />
```

Comment:
@shelegdmitriy I think this change is the culprit for the icons not loading anymore.

Reply:
@calebjacob after https://github.com/near/near-discovery/pull/1150 feel free to test on this preview and merge this  current PR